docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+    simulator-api:
+        build: ./minitwit-backend
+        ports:
+            - "5001:5001"
+        volumes:
+            - /tmp/minitwit.db:/tmp/minitwit.db
+        command: npm run simulator
+
+    custom-api:
+        build: ./minitwit-backend
+        ports:
+            - "5000:5001"
```

Comment:
Hvorfor skal der ikke stå `5000:5000`? 😬

Reply:
To processer kan ikke køre på samme port.